packages/merge-conflicts/src/browser/merge-conflict.ts

```diff
@@ -35,20 +35,20 @@ export interface MergeConflictCommandArgument {
 }
 
 export namespace MergeConflictsCommands {
-    const MERGE_CONFLIC_PREFIX = 'Merge Conflict';
+    export const MERGE_CONFLICT_PREFIX = 'Merge Conflict';
```

Comment:
Why do we need to export this one?

Reply:
I included the `export` so that others (extenders) would be able to access the prefix and be able to categorize their commands with existing ones if they wanted to. Do you believe we should not export?

Reply:
ah ok maybe you would have another PR where `MERGE_CONFLICT_PREFIX ` is referenced ?

Reply:
Exactly, let's say I'm an extender who wishes to use the **prefix** because they are contributing commands to the same category. Before the pull request, they'd have to duplicate and hard code the value while if it is **exported** they can simply reference it.